pkg/util/gardener/util_test.go

```diff
@@ -6,12 +6,12 @@ import (
 )
 
 var _ = Describe("uti/gardener", func() {
-	Describe("IsShootCPNamespace", func() {
+	Describe("IsShootNamespace", func() {
 		It("should work expected on certain predefined values", func() {
-			Expect(IsShootCPNamespace("shoot--my-shoot")).To(BeTrue())
-			Expect(IsShootCPNamespace("shoot-my-shoot")).To(BeFalse())
-			Expect(IsShootCPNamespace("")).To(BeFalse())
-			Expect(IsShootCPNamespace("shoot--my--shoot")).To(BeTrue())
+			Expect(IsShootNamespace("shoot--my-shoot")).To(BeTrue())
+			Expect(IsShootNamespace("shoot-my-shoot")).To(BeFalse())
```

Comment:
This must be true as @ialidzhikov  also mentioned in https://github.com/gardener/gardener-custom-metrics/pull/7/files#r1528132711

Reply:
Thanks!